content/tutorials/essentials/3-manifolds.py

```diff
@@ -204,19 +209,22 @@
 # beta_go = la.Fixed(pd.Series([1., 100.], index = adata_oi.obs["gene_overexpressed"].cat.categories), label = "beta")
 
 # %% [markdown]
-# This nudging is performed by an appropriate prior distribution. We choose a beta distribution here because
+# This nudging is performed by an appropriate prior distribution. A beta distribution becomes very handy here, indeed depending on the parameters it can either be a uniform  distribution or can be concentrated at any value we want! So we will now set the prior of the differentiation as a beta distribution for which its two parameters alpha and beta will depend on if it's a control of *Myod1* overexpressing cell. 
+
+# %% [markdown]
+# Note that a $\beta$(1,1) is a uniform, and a $\beta$(1,100) is very concentrated at 0. 
 
 # %%
 import pandas as pd
 
 differentiation_p = la.distributions.Beta(
-    beta=la.Fixed(
+    alpha=la.Fixed(
```

Comment:
Should check whether this is really correct. Don't we want a high beta for mCherry cells? I.e. green curve:
![image](https://user-images.githubusercontent.com/5727377/138050791-f41d7811-2f22-4406-9a67-1a8694b0e3cf.png)